archive/terraform/archive_api_swagger.json

```diff
@@ -87,12 +87,18 @@
                 "application/json": "{\"id\": \"$input.path('$.id')\"}"
               }
             },
-            ".*[BadRequest].*": {
+            ".*[${error_bad_request}].*": {
               "statusCode": "400",
               "responseTemplates": {
                 "application/json": "#set($inputRoot = $input.path('$')){\"message\" : \"$inputRoot.errorMessage\"}"
               }
             },
+            ".*[${error_method_not_allowed}].*": {
+              "statusCode": "405",
```

Comment:
The broad match here is partly to get it working, we could now anchor it ^${error_bad_request}.*? Might prevent something unexpected if someone breaks the conventions for the error strings.